troi/playlist.py

```diff
@@ -68,6 +67,15 @@ def _serialize_to_jspf(playlist, created_for=None, track_count=None):
                     "artist_identifiers": artist_mbids,
                 }
             }
+
+        if e.release is not None:
+            if e.release.name != "":
+                track["album"] = e.release.name
+
+            if e.release.mbid != "":
+                track["extension"]["album"][PLAYLIST_TRACK_EXTENSION_URI]["release_identifier"] = \
```

Comment:
This doesn't look right.
Should it be this?
```suggestion
                track["extension"][PLAYLIST_TRACK_EXTENSION_URI]["release_identifier"] = \
```

Reply:
That might explain why it wasn't working.  I ended up giving up for the day, lol.